nrf_802154/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2017 - 2021, Nordic Semiconductor ASA
```

Comment:
I thought we were to use Nordic 5-clause license in NCS and re-licensing to BSD 3-clause in Zephyr. Is using BSD here intentional?

Reply:
I have put BSD 3-clause at the `nrf_802154` root and Nordic 5-clause at `nrf_802154/sl/sl/lib`. A similar approach was taken here: https://github.com/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver

Reply:
I thought we planned to change it. But it's a minor. We can discuss it later